scripts/skills/effects/rf_between_the_eyes_effect.nut

```diff
@@ -0,0 +1,72 @@
+this.rf_between_the_eyes_effect <- ::inherit("scripts/skills/skill", {
+	m = {
+		MeleeSkillToHeadshotChancePct = 0.5
+		HeadshotDamageMult = 1.5
+	},
+	function create()
+	{
+		this.m.ID = "effects.rf_between_the_eyes";
+		this.m.Name = "Between the Eyes";
+		this.m.Description = "This character is prepared to land his next attack right between the target\'s eyes.";
+		this.m.Icon = "ui/perks//perk_rf_between_the_eyes.png";
+		this.m.Overlay = "rf_between_the_eyes_effect";
+		this.m.Type = ::Const.SkillType.StatusEffect;
+		this.m.IsRemovedAfterBattle = true;
+	}
+
+	function getTooltip()
+	{
+		local ret = this.skill.getTooltip();
+		local chanceStr;
+		if (::MSU.isEqual(this.getContainer().getActor(), ::MSU.getDummyPlayer()))
+		{
+			chanceStr = "chance equal to " + ::MSU.Text.colorizePct(this.m.MeleeSkillToHeadshotChancePct) + " of [Melee Skill|Concept.MeleeSkill]"
+		}
+		else
+		{
+			chanceStr = ::MSU.Text.colorizeValue(this.getHeadshotChanceAdd(), {AddSign = true, AddPercent = true}) + " chance"
+		}
+		ret.push({
+			id = 10,
+			type = "text",
+			icon = "ui/icons/chance_to_hit_head.png",
+			text = ::Reforged.Mod.Tooltips.parseString("The next melee attack has a " + chanceStr + " to hit the head and deals " + ::MSU.Text.colorizeMult(this.m.HeadshotDamageMult) + " damage on a such a hit")
+		});
+
+		return ret;
+	}
+
+	function onAnySkillUsed( _skill, _targetEntity, _properties )
+	{
+		if (this.isSkillValid(_skill))
+		{
+			_properties.HitChance[::Const.BodyPart.Head] += this.getHeadshotChanceAdd();
+			_properties.DamageAgainstMult[::Const.BodyPart.Head] *= this.getHeadshotDamageMult();
```

Comment:
The existing vanilla effects, which utilize `DamageAgainstMult` only ever change it additively.
See `brute_trait.nut` or `chop.nut`

If you break that convention only at this place then there should be a comment here explaining why

Reply:
I don't really know why vanilla went the additive route. But perhaps it was because they wanted each of these traits to apply a fixed effect instead of multiplying each other's effects. I'll add a comment that we break the vanilla convention and apply multiplicative here.